app/serializers/articles_show_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class ArticlesShowSerializer < ActiveModel::Serializer
-  attributes :id, :title, :teaser, :body, :date, :author, :category
+  attributes :id, :title, :teaser, :body, :date, :author, :category, :image
```

Comment:
And this? :3

Reply:
```rb
def image  
  Rails.env.test? ?
    ActiveStorage::Blob.service.path_for(image.key) 
    : 
    image.service_url(expires_in: 1.hour, disposition: 'inline') 
end
```

Reply:
Perhaps we need something like this

Reply:
what goin on here?

Reply:
This Amazon remote storage works in development environment but not in test environment. So when we run tests we don't have image url, so the line ```image.service_url(expires_in: 1.hour, disposition: 'inline') ``` errors out. We can check what environment we are in and make serializer return different stuff in different environments. So the tests are green and we get our urls when deployed

Reply:
Awesum stuff